audio/Audio.cc

```diff
@@ -215,7 +215,7 @@ Audio* Audio::get_ptr() {
 Audio::Audio() {
 	assert(self == nullptr);
 
-	string s;
+	string s, newval;
```

Comment:
Why not declare newval where it is first used (line 234)?

Reply:
I put the declaration in first before using it